assets/js/modules/pagespeed-insights/index.js

```diff
@@ -20,3 +20,17 @@
  * Internal dependencies
  */
 import './datastore';
+import { fillFilterWithComponent } from '../../util';
+import { SettingsMain as PageSpeedInsightsSettings } from './settings';
+
+/**
+ * WordPress dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+
+/**
+ * Add components to the settings page.
+ */
+addFilter( 'googlesitekit.ModuleSettingsDetails-pagespeed-insights',
+	'googlesitekit.PageSpeedInsightsModuleSettingsDetails',
+	fillFilterWithComponent( PageSpeedInsightsSettings ), 10 );
```

Comment:
Formatting, for consistency, also removing the default priority from `addFilter` as we aren't using this in any of the other new components. Let's only add it if we need it.

```suggestion
addFilter(
	'googlesitekit.ModuleSettingsDetails-pagespeed-insights',
	'googlesitekit.PageSpeedInsightsModuleSettingsDetails',
	fillFilterWithComponent( PageSpeedInsightsSettings )
);
```

Reply:
Updated